snakemake_wrapper_utils/snakemake.py

```
@@ -44,4 +49,29 @@ def get_format(path):
+
+    mapping must be a dict of {out_tag: source_path}.
+
+    Example:
+        mapping = {"tsv": "/tmp/tmp98723489/results/out.tsv"}
+
+    This moves /tmp/tmp98723489/results/out.tsv to snakemake.output["tsv"],
+    redirecting stdout/stderr to snakemake.log.
+    """
+
+    cmds = list()
+    for out_tag, tool_out_name in mapping.items():
```

Ruff (0.12.2)

Loop control variable `tool_out_name` not used within loop body

Rename unused `tool_out_name` to `_tool_out_name`

(B007)